src/context/MarFashionProvider.tsx

```diff
@@ -66,16 +65,11 @@ const MarFashionProvider: FC<Props> = ({ children }) => {
   const Logout = () => {
     removeCookie("user")
     removeCookie("token")
-    setIsLogin(false);
     router.push('/', undefined, { shallow: true })
   }
 
-  const login = () => {
-    setIsLogin(true);
-  }
-
   return (
-    <MarFashionContext.Provider value={{ user, setUser, Login, Logout, login, isLogin }}>
+    <MarFashionContext.Provider value={{ user, setUser, Login, Logout }}>
       {children}
     </MarFashionContext.Provider>
   )
```

Comment:
export useContext in this file `useContext(MarFashionContext)`, so in other file only call this
for example
```
export useMarContext = useContext(MarFashionContext);
```

and dont export this
```
const MarFashionContext
```

Reply:
done